RecoLocalMuon/Configuration/python/RecoLocalMuon_EventContent_cff.py

```diff
@@ -22,7 +22,7 @@
 )
 # AOD content
 RecoLocalMuonAOD = cms.PSet(
-    outputCommands = cms.untracked.vstring(
+    outputCommands = cms.untracked.vstring( 
         'keep *_dt4DSegments_*_*', 
         'keep *_dt4DCosmicSegments_*_*',
         'keep *_cscSegments_*_*', 
```

Comment:
BTW, both DT segments and CSC segments are already available in AOD.
It would be nice to see some slides/material showing that segments do not work.

Reply:
@slava77 For the record, we checked in simulation that rechits provides much better signal-background discrimination. A more detailed discussion is also documented in the AN of EXO-20-015, section 4.1 

![image](https://user-images.githubusercontent.com/12798013/121974198-85ba9580-cd44-11eb-9e9d-eff425533388.png)